roles/common/tasks/main.yml

```diff
@@ -0,0 +1,75 @@
+---
+# vm configs
+- name: disable THP
+  template: 
+    src: templates/disable-thp.service.j2
+    dest: /etc/systemd/system/disable-thp.service
+
+- name: reload and enable 
+  systemd: 
+    name: disable-thp.service
+    state: started
+    enabled: true
+    daemon_reload: true
```

Comment:
This doesn't need to be a service file. We can use `lineinfile` to write to `/sys/kernel/mm/transparent_hugepage/enabled` and `/sys/kernel/mm/transparent_hugepage/defrag` directly.

Can we rework this using `lineinfile`

Reply:
I used a service file based on Couchbase recommended best practice, which resumes on a reboot vs falling back to the kernel default.